great_expectations/checkpoint/util.py

```diff
@@ -179,231 +166,6 @@ def send_email(  # noqa: C901, PLR0913
         return "success"
 
 
-def get_substituted_validation_dict(
```

Comment:
All of the deleted utils have to do with legacy checkpoints - the remainder of the functions here are for actions and should be moved into that file